spec/Ranges.tex

```diff
@@ -475,8 +475,10 @@ \subsection{Range Assignment}
 \item An implicit conversion is allowed from \chpl{idxType} of the source range
        to \chpl{idxType} of the destination range type,
 \item the two range types have the same \chpl{boundedType}, and
-\item either the destination range is stridable or the source range's
-      stride is 1.
+\item either the destination range is stridable or the source range
+      is not stridable.
+\item Or an explicit conversion is used to make the source range match the
+      type of the destination range.
```

Comment:
What if I reduce it to two bullets by combining the first 3 into one?

Range assignment is legal when:

* Either an explicit conversion is used to make the source range match the type of the destination range, 

* or an implicit conversion is allowed from idxType of the source range to idxType of the destination range type and the two range types have the same boundedType, and either the destination range is stridable or the source range is not stridable.